### PR TITLE
[WiP] adding Energy dataModels

### PR DESCRIPTION
**Pull request by stbnrivas**
hello, I would like add some datamodels to Fiware about:
- battery
- batteryStatus
- consumptionMeasurement

Thanks in advanced

### CODE REVIEW

File: specs/Energy/Battery/example.jsonld [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "id": "urn:ngsi-ld:Battery:santander:energy:bat-ac-d95372df39",
+  "type": "Battery",
+  "source": {
```
Comment (jmcanterafonseca) [inline specs/Energy/Battery/example.jsonld]: check the NGSI-LD representation of other entities, this is totally incorrect

Comment (stbnrivas) [reply specs/Energy/Battery/example.jsonld]: ok

**Comment by jmcanterafonseca** [comment]
still CI broken

**Comment by stbnrivas** [comment]
sorry for the delay,  I am thinking about id of examples NGSIv2 linked at main README.md : [https://github.com/FIWARE/data-models/blob/master/specs/Weather/WeatherObserved/example.json](https://github.com/FIWARE/data-models/blob/master/specs/Weather/WeatherObserved/example.json)
```json
{
  "id": "Spain-WeatherObserved-Valladolid-2016-11-30T07:00:00.00Z"
}
```
id like this, create a single entity for every second, hour or day??, the point is the context should rewrite entities but doesn't, create a new entity for every time interval ....

Can I use for example:
1. into `example.json` id like `${ype}:${city}:${vertical}:${uuid}`
2. into `example-normalized.json` same type id `${ype}:${city}:${vertical}:${uuid}`
3. into `example-normalized-ld.jsonld` `urn:ngsi-ld:${ype}:${city}:${vertical}:${uuid}`

Thanks in advanced

**Comment by jmcanterafonseca** [comment]
CI still broken. please check

**Comment by stbnrivas** [comment]
Sorry, it seems that I will need some help @jmcanterafonseca 

```
can't resolve reference 
https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons 
from id 
https://fiware.github.io/dataModels/specs/Energy/Battery/schema.json#
...
Examples cannot be validated since validation function cannot be computed. Probably not all schemas can be resolved correctly (check schema errors)

```


well it looks like the CI is searching can't find common-schema.json ? only for my models

**Comment by jmcanterafonseca** [comment]
be careful now the URI for the common schemas should include data-models
instead of dataModels ...

**Comment by stbnrivas** [comment]
thank you so much!! I almost got it, now only I have only warnings but I don't understand CI explanation

**Comment by jmcanterafonseca** [comment]
the build is now broken due to several markdown warnings. once we fix it
please rebase and check CI.

**Comment by stbnrivas** [comment]
thank you so much!!

**Comment by jmcanterafonseca** [comment]
please rebase this and rerun CI, everything should be fine

**Comment by jmcanterafonseca** [comment]
CI still broken.

**Comment by jmcanterafonseca** [comment]
@stbnrivas

**Comment by stbnrivas** [comment]
I thought that I should not modify ngsi-ld_faq.md in this pull request because it belongs to another context.

/home/travis/build/FIWARE/data-models/specs/ngsi-ld_faq.md

  78:32  error  https://schema.lab.fiware.org/ld/context is dead. (301 Moved Permanently)  no-dead-link

Just let me know and I will change. Thank you

**Comment by jmcanterafonseca** [comment]
you need to rerun the CI and check now

**Comment by albertoabellagarcia** [comment]
Moved this PR a smart data models Battery 
https://github.com/smart-data-models/dataModel.Battery

**Pull request closed without merging**